src/Symfony/Component/Ldap/Adapter/AdapterInterface.php

```diff
@@ -32,7 +32,7 @@ public function getConnection();
      *
      * @return QueryInterface
      */
-    public function createQuery($dn, $query, array $options = []);
+    public function createQuery(string $dn, string $query, array $options = []);
```

Comment:
We can remove the `@param` annotations here and from a quite a few of other methods as well.

Reply:
fixed